src/setup.c

```diff
@@ -336,7 +336,9 @@ static void setup_adc2(void)
  * - Configure the base time (no clock division ratio, no aligned mode,
  *   direction up).
  * - Set clock division, prescaler and period parameters to get an update
- *   event with a frequency of 100 KHz.
+ *   event with a frequency of 16 KHz. 16 interruptions by ms, 4 sensors with
+ *   4 states. Update event frequency = timer_clock/((preescaler + 1) * (period
+ * + 1))
```

Comment:
I'd rather use equation formatting here. I think you can use LaTeX formulas.

```latex
\f$frequency = \frac{timerclock}{(preescaler + 1) (period + 1)}\f$
```

Reply:
Good idea, I didn't try it with Doxygen... but I believe you :)